src/engraving/playback/playbackeventsrenderer.cpp

```diff
@@ -219,9 +219,9 @@ void PlaybackEventsRenderer::renderMetronome(const Score* score, const int tick,
     static ArticulationMap emptyArticulations;
 
     BeatType beatType = score->tick2beatType(Fraction::fromTicks(tick));
-    pitch_level_t eventPitchLevel = pitchLevel(PitchClass::A, 4);
+    pitch_level_t eventPitchLevel = pitchLevel(PitchClass::E, 5);
     if (beatType == BeatType::DOWNBEAT && tick == 0) {
-        eventPitchLevel = pitchLevel(PitchClass::B, 4);
+        eventPitchLevel = pitchLevel(PitchClass::F, 5);
     }
```

Comment:
It's better to use the high sound on the downbeat. Also, not your code, but the `tick == 0` condition is undesirable here. It means to do this only on the "first beat of the score" rather than "first beat of every bar".

I recommend changing those lines to:

```C++
    BeatType beatType = score->tick2beatType(Fraction::fromTicks(tick));
    pitch_level_t eventPitchLevel = beatType == BeatType::DOWNBEAT
                                    ? pitchLevel(PitchClass::E, 5) // high wood block
                                    : pitchLevel(PitchClass::F, 5); // low wood block
```

This will make the metronome sound more like it did in MuseScore 3.